### PR TITLE
Sofia/common collections/strings

### PR DESCRIPTION
**Pull request by sofiiako**

### CODE REVIEW

File: Common Collections/Strings/HashMaps/task.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+```rust
+    use std::collections::HashMap;
+
+    let text = "hello world wonderful world";
+
+    let mut map = HashMap::new();
+
+    for word in text.split_whitespace() {
+        let count = map.entry(word).or_insert(0);
+    *count += 1;
```
Comment (Geravant) [inline Common Collections/Strings/HashMaps/task.md:248]: I believe the tabulation is missing here for the *count +=1; line

File: Common Collections/Strings/Declaring a Vector/tests/tests.rs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+#[test]
+fn test_array_and_vec_similarity() {
+    let (a, v) = array_and_vec();
+    assert_eq!(a, v[..]);
+}
+
+
+
+//fn is_correct() {
```
Comment (Geravant) [inline Common Collections/Strings/Declaring a Vector/tests/tests.rs:17]: We can remove this, as we are not using it after all

File: Common Collections/Strings/Loop Over a Vector/task-info.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- name: Cargo.toml
+  visible: true
+- name: tests/tests.rs
+  visible: false
+- name: src/lib.rs
+  visible: true
+  placeholders:
+  - offset: 83
+    length: 8
+    placeholder_text: Fill this up so that each element in the Vec `v` is multiplied
```
Comment (Geravant) [inline Common Collections/Strings/Loop Over a Vector/task-info.yaml:14]: Let's add //at the beginning of the placeholder text, so it won't be painted red by the Idea at the beginning of the exercise

File: Common Collections/Strings/Declaring a Vector/task-info.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+- name: Cargo.toml
+  visible: true
+- name: tests/tests.rs
+  visible: false
+- name: src/lib.rs
+  visible: true
+  placeholders:
+  - offset: 108
+    length: 11
+    placeholder_text: Declare your vector here
```
Comment (Geravant) [inline Common Collections/Strings/Declaring a Vector/task-info.yaml:14]: Let's add //at the beginning of the placeholder text, so it won't be painted red by the Idea at the beginning of the exercise

File: Common Collections/Strings/Declaring a HashMap/task-info.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+- name: Cargo.toml
+  visible: true
+- name: tests/tests.rs
+  visible: false
+- name: src/lib.rs
+  visible: true
+  placeholders:
+  - offset: 101
+    length: 15
+    placeholder_text: Declare your hash map here
```
Comment (Geravant) [inline Common Collections/Strings/Declaring a HashMap/task-info.yaml:14]: Let's add //at the beginning of the placeholder text, so it won't be painted red by the Idea at the beginning of the exercise

File: Common Collections/Strings/Declaring a HashMap/task-info.yaml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  - offset: 215
+    length: 85
+    placeholder_text: Put more fruits in your basket here!
```
Comment (Geravant) [inline Common Collections/Strings/Declaring a HashMap/task-info.yaml:17]: Let's add //at the beginning of the placeholder text, so it won't be painted red by the Idea at the beginning of the exercise

File: Common Collections/Strings/Declaring a HashMap/task.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+The key represents the name of the fruit and the value represents
+how many of that particular fruit is in the basket. You have to put
+at least three different types of fruits (e.g apple, banana, mango)
+in the basket and the total count of all the fruits should be at
+least five.
+
+Make yur code compile and pass the tests!
+
+<div class="hint"> Take a look at the return type of the function to figure out
+  the type for the `basket`.</div></li>
```
Comment (Geravant) [inline Common Collections/Strings/Declaring a HashMap/task.md:13]: The tag does not work here, we should probably change the `` to an HTML tag

File: Common Collections/Strings/Modifying a HashMap/src/main.rs [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+        let mut basket = HashMap::<modifying_a_hashmap::Fruit, u32>::new();
+        basket.insert(modifying_a_hashmap::Fruit::Apple, 4);
+        basket.insert(modifying_a_hashmap::Fruit::Mango, 2);
+        basket.insert(modifying_a_hashmap::Fruit::Lichi, 5);
+
+        basket
+    }
+    let mut basket = get_fruit_basket();
+    let b = fruit_basket(&mut basket);
+    println!("Basket: {:?}", b);
```
Comment (Geravant) [inline Common Collections/Strings/Modifying a HashMap/src/main.rs:17]: We probably should modify it, it does not currently print anything, so it is unclear what modifications were made by the student

**Review by Geravant — CHANGES_REQUESTED** [review]

**Comment by Geravant** [comment]
Great job!
The exercises seem very cool, just a few tweaks, mostly formatting stuff.

**Comment by sofiiako** [comment]
@Geravant addressed the issues

**Review by Geravant — APPROVED** [review]

**Pull request merged**